### PR TITLE
GW-25: Develop partners section in home page

### PR DESCRIPTION
**Pull request by chrishadrian**
- Home background is changed to `FFFDF5`
- Modified activities and about sections
- Imported partners' images to the codebase
- Imported `react fast marquee` and `react responsive` libraries

Preview: https://deploy-preview-14--gisaubcv3.netlify.app

### CODE REVIEW

**Review by jenaarianto — APPROVED** [review]

File: src/pages/home/components/Activities/Activities.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Box } from "@mui/material";
-import { activities, smiley } from "./constants";
+import { activities, smiley } from "../Activities/constants";
```
Comment (Julian-UBC) [inline src/pages/home/components/Activities/Activities.js:3]: Why did you capitalize the folder name? Any reason?

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
Next time don't forget to put `feature/` in front of your branch name

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
Include descriptions of what you have implemented and/or changed in this PR

**Comment by chrishadrian** [comment]
Last commit should be a merge commit. Ignore the commit message.

**Review by Julian-UBC — APPROVED** [review]

**Pull request merged**